addon/mixins/resolver.js

```diff
@@ -469,13 +481,8 @@ export default Mixin.create({
 	 * @return String
 	 */
 	findModule(namespace, type, name) {
-		const str = makeToString(namespace, type, name);
 		const resolvers = A([this.findModuleByMain, this.findModuleByType, this.findModuleByPod]);
 
-		if (this._moduleCache[str]) {
```

Comment:
Why did you remove this? It is meant to be a cache to prevent resolve same modules again.

Reply:
The Ember registry always answer  before the resolver:
https://github.com/emberjs/ember.js/blob/master/packages/%40ember/-internals/container/lib/registry.ts#L789